app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        today = datetime.date.today()
+        if "vaccine" in visitor:
+            expiration_date = visitor["vaccine"]["expiration_date"]
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("NotVaccinatedError")
+        else:
+            expiration_date = visitor["vaccine"]["expiration_date"]
```

Comment:
Use method `get` to avoid KeyError
```suggestion
            expiration_date = visitor["vaccine"].get("expiration_date"])
```